### PR TITLE
 login page

### PR DESCRIPTION
**Pull request by Aksh-Bansal-dev**

### CODE REVIEW

**Comment by Aksh-Bansal-dev** [comment]
I think it will be better not to squash both the commits in this PR.

File: ui/src/pages/Homepage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const Homepage: React.FC = () => {
```
Comment (Samy-33) [inline ui/src/pages/Homepage/index.tsx:1]: Can be renamed to `Home`

File: ui/src/pages/Login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import {logo} from '../../assets/logo';
+import styles from './styles.module.css';
+
+const Login: React.FC = () => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.card}>
+        <img className={styles.logo} src={logo} alt="logo" />
+        <div className={styles.heading}>Gymkhana</div>
+        <button className={styles.btn}>SIGN IN</button>
```
Comment (Samy-33) [inline ui/src/pages/Login/index.tsx:10]: Are we planning to add the `icon` on the right later?

File: ui/src/pages/Login/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  background-color: var(--primary-bg);
+  height: 60vh;
+  width: 22vw;
+  border-radius: 40px;
+  filter: drop-shadow(10px 10px 30px rgba(0, 0, 0, 0.25));
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 10vh 0 0 0;
```
Comment (Samy-33) [inline ui/src/pages/Login/styles.module.css:21]: Can be converted to `filter-top`?

File: ui/src/pages/Login/styles.module.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  margin-top: 5vh;
+  width: 100%;
+  color: var(--secondary-color);
+  background-color: var(--primary-color);
+  height: 10vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 0 0 40px 40px;
+}
```
Comment (Samy-33) [inline ui/src/pages/Login/styles.module.css:56]: Somehow this style leaves space at the bottom.

image

File: ui/src/index.css [hunk 5]
```diff
@@ -1,13 +1,20 @@
...
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-bg: #ffffff;
+  --primary-color: #d81f26;
+  --primary-text: #304455;
+  --secondary-bg: #829bb0;
+  --secondary-color: white;
```
Comment (Samy-33) [inline ui/src/index.css:19]: Can we group these by `bg`, `color` and `text`?

**Review by Samy-33 — CHANGES_REQUESTED** [review]
We should keep `images` inside `assets`.

File: ui/src/pages/Login/styles.module.css (hunk 3 above)
Comment (Aksh-Bansal-dev) [reply ui/src/pages/Login/styles.module.css:21]: Can you explain this one? What do you mean by `filter-top`?

Comment (Samy-33) [reply ui/src/pages/Login/styles.module.css:22]: My bad, I meant `padding-top`.

**Review by Samy-33 — DISMISSED** [review]
LGTM, 

Just a small change. The background is repeated rn.
image

Can we change it to stretch to fill the whole screen?

**Review by Samy-33 — APPROVED** [review]
LGTM

**Pull request merged**